dbsim/db_server_state.hpp

```diff
@@ -40,6 +40,7 @@ namespace db
                 mutex_,
                 cond_,
                 server_service,
+                NULL,
```

Comment:
Could use `nullptr` here (dbsim is compiled with -std=c++14).

Reply:
Changed.